src/libraries/System.Net.Http/src/System/Net/Http/EmptyContent.cs

```diff
@@ -22,14 +22,14 @@ protected internal override bool TryComputeLength(out long length)
 
         protected override Task SerializeToStreamAsync(Stream stream, TransportContext? context, CancellationToken cancellationToken) =>
             cancellationToken.IsCancellationRequested ? Task.FromCanceled(cancellationToken) :
-            SerializeToStreamAsync(stream, context);
+            SerializeToStreamAsync(stream, context, cancellationToken);
```

Comment:
This looks like a bug in the analyzer.  This is changing the code to call itself and will result in an infinite loop or stack overflow.  This change should be reverted.

Reply:
Thanks, good catch.

Reply:
This case is going to be fun to fix 😄

Reply:
@Jozkee just addressed (or was going to address if he hasn't already) a similar issue in one of his analyzers.